config.yml

```diff
@@ -55,8 +55,8 @@ kin:
 
 # [XTZ] Tezos: https://tezos.com
 tezos:
-  api: https://api1.tzscan.io/v3
-  rpc: https://mainnet-node.tzscan.io
+  api: https://api6.dunscan.io
+  rpc: https://mainnet-node.dunscan.io
```

Comment:
dunscan is a different blockchain no?

Reply:
If I got it right they now support Tezos as well
![image](https://user-images.githubusercontent.com/16366057/67259805-59486a80-f4ca-11e9-8c18-07319ce5bb86.png)